### PR TITLE
Cleanup L0 image pkr.hcl and kkm packer tests

### PR DESCRIPTION
**Pull request by gnode1**
run kkm test in packer driven VMs.

### CODE REVIEW

File: .github/workflows/km-ci-workflow.yaml [hunk 1]
```diff
@@ -62,7 +62,6 @@ env:
   SP_APPID: ${{ secrets.SP_APPID }}
   SP_PASSWORD: ${{ secrets.SP_PASSWORD }}
   SP_TENANT: ${{ secrets.SP_TENANT }}
-  GITHUB_TOKEN: ${{ secrets.FULL_GITHUB_ACCESS_TOKEN }} # the one in secrets covers more than github.token
```
Comment (paulpopelka) [inline .github/workflows/km-ci-workflow.yaml:65]: This is removed because the github repository is now public?

**Comment by gnode1** [comment]
Exactly

**Review by paulpopelka — APPROVED** [review]
This area is at the limits of my understanding.
Someone else should also review.

File: tests/packer/scripts/km-test.sh [hunk 2]
```diff
@@ -32,6 +32,7 @@ echo LOCATION=${LOCATION?Please set LOCATION env var to KM source dir where the
 # Useful debug info (sans secrets)
 env # | grep -v SP_ | grep -v GITHUB_TOKEN
 
+rm -rf km
 # we only need KKM repo and Make system so not pulling in all submodules
 echo Cloning KM repo. branch: ${SRC_BRANCH}
 git clone https://$GITHUB_TOKEN@github.com:/kontainapp/km -b ${SRC_BRANCH}
```
Comment (sv641) [inline tests/packer/scripts/km-test.sh:38]: GITHUB_TOKEN is removed.

**Review by sv641 — APPROVED** [review]

**Pull request merged**